README.md

```diff
@@ -53,7 +53,7 @@ There are 3 major segments of the applications API
 	}
 	```
 
-	and will return a response of format
+	and will return a response in the format
```

Comment:
**Suggestion:** Correct the grammar in line 56 from 'a response in the format' to 'a response in the following format'.
```suggestion
 and will return a response in the following format
```